lib/sacastats_web/views/character_view.ex

```diff
@@ -1,3 +1,114 @@
 defmodule SacaStatsWeb.CharacterView do
   use SacaStatsWeb, :view
+
+  alias SacaStats.{Events, Session}
+
+  require Logger
+
+  def pretty_session_summary(assigns, session) do
+    login_time = prettify_timestamp(session.login.timestamp)
+    logout_time = prettify_timestamp(session.logout.timestamp)
+    session_duration = prettify_duration(session.login.timestamp, session.logout.timestamp)
+
+    ~H"""
+    <div class="poll-item text-dark">
+      <%= login_time %> → <%= logout_time %>
+      <br>
+      <%= session_duration %>
+    </div>
+    """
+  end
+
+  def prettify_timestamp(:current_session), do: "Current Session"
+
+  def prettify_timestamp(timestamp) do
```

Comment:
Oh and I forgot to mention if you would like to just import/call
`import { addFormatsToPage } from "/js/formats.js";`

You would be able to switch the default UTC date-time to the clients local date time zone. But you would need to surround just the date time with a classed element of `date-time` like so
`<p class="d-inline date-time">{UTC DATE TIME HERE}</p>`

Reply:
I tried adding this but couldn't get it to work, so I'll let you put this together when it comes to making the page pretty (you'll do a much better job than me anyway)